src/ai/backend/manager/models/base.py

```diff
@@ -1178,7 +1186,13 @@ async def populate_fixture(
             # skip reserved names like "__mode"
             continue
         assert not isinstance(rows, str)
-        table: sa.Table = getattr(models, table_name)
+
+        # TODO: Remove this try statement after all the table upgrade is done
+        try:
+            table: sa.Table = getattr(models, table_name)
+        except AttributeError:
+            table = await load_table(engine, table_name)
+
```

Comment:
Could you describe more about what this block is for?

Reply:
In this block, the below tables data will be loaded by the `table: sa.Table = getattr(models, table_name)`.

```
domains,
user_resource_policies,
project_resource_policies,
groups,
scaling_groups,
sgroups_for_domains,
users,
association_groups_users,
keypair_resource_policies,
keypairs,
users,
resource_presets
```

You can see the specified tables are defined like the below form.

```
domains = sa.Table(
...
)
user_resource_policies = sa.Table(
...
)
project_resource_policies = sa.Table(
...
)

...
``` 

That's why they can be loaded through `getattr(models, table_name)`.

*However, we did not define the ContainerRegistry table in this way.* 

Therefore, since I cannot retrieve the table information through `getattr(models, table_name)`, I loaded and accessed the DB directly to load the table information.

There might be a better way to load table information that I am not aware of.
If you know any, I would appreciate it if you could share it with me.

Reply:
Resolved in https://github.com/lablup/backend.ai/pull/1917/commits/ad881b22223cf90f0f3f0366fbe665721766a52f